packages/bahmni-design-system/src/organisms/dataTable/styles/DataTable.module.scss

```
@@ -0,0 +1,116 @@
+    left: auto;
+  }
+}
+
+.visuallyHidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)